cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/view/AwsInstanceView.java

```diff
@@ -74,15 +74,7 @@ public String getKmsKey() {
         return instanceTemplate.getStringParameter("key");
     }
 
-    public Double getSpotPrice() {
-        Object sv = instanceTemplate.getParameter("spotPrice", Object.class);
-        if (sv instanceof Double) {
-            return (Double) sv;
-        } else if (sv instanceof Integer) {
-            return ((Number) sv).doubleValue();
-        } else if (sv instanceof String) {
-            return Double.valueOf((String) sv);
-        }
-        return null;
+    public Integer getSpotPercentage() {
+        return instanceTemplate.getParameter("spotPercentage", Integer.class);
```

Comment:
isn't it just percentage?

Reply:
on api level it is, but as internal parameter accompanying the instanceTemplate sitting in a map it is more clear, I guess

Reply:
ok